lib/marked.js

```diff
@@ -1084,13 +1084,33 @@ Parser.prototype.tok = function() {
  * Helpers
  */
 
+var escapeTest = /[&<>"']/;
+var escapeReplace = /[&<>"']/g;
+var replacements =
+  {
+    '&': '&amp;',
+    '<': '&lt;',
+    '>': '&gt;',
+    '"': '&quot;',
+    "'": '&#39;'
+  };
+
+var escapeTestNoEncode = /(?:[<>"']|&(?!#?\w+;))/;
```

Comment:
There's no need to use grouping to wrap the whole thing

Reply:
Thank you, I fixed it.

Reply:
Y'all are awesome! Thank you.